rpm.el8.spec

```diff
@@ -0,0 +1,39 @@
+#
+# spec file for package cloud-init-vmware-guestinfo
+#
+
+#################################################################################
+# common
+#################################################################################
+Name:           cloud-init-vmware-guestinfo
+Version:        1.1.0
+Release:        1.el8
+Summary:        A cloud-init datasource that uses VMware GuestInfo
+License:        Apache2
+Requires:       cloud-init python3-netifaces
+Group:          Applications/System
+BuildArch:      noarch
+
+#################################################################################
+# specific
+#################################################################################
+%description
+A cloud-init datasource that uses VMware GuestInfo
+
+%prep
+
+%build
+
+%install
+mkdir -p %{buildroot}/etc/cloud/cloud.cfg.d
+mkdir -p %{buildroot}/usr/lib/python3.6/site-packages/cloudinit/sources
+cp 99-DataSourceVMwareGuestInfo.cfg %{buildroot}/etc/cloud/cloud.cfg.d/99-DataSourceVMwareGuestInfo.cfg
```

Comment:
Please pull in my updated Makefile and Dockerfile to validate the RPM. Either as commits to your PR or as your own commits. I don't care if you take that code. Anyway, I'd prefer we update these lines to match what is in the el7 spec. If you don't want to for some reason, please let me know.

Reply:
I added your commit (c767ab9983f509d24e876c7637b1b4c21bab5137) on top of mine. I was originally thinking you'd do a separate PR or merge it after this would be accepted but this is fine by me as well.

Reply:
Regarding the difference in those lines, I don't remember the exact reason for the change and I don't have any strong preference of keeping them the way they're on my branch.

I can change them and if no apparent reason pops up in my tests, they can be that way

Reply:
Eh, not worth it. Thanks for being open to it!